apps/apps.humanatlas.io/src/app/app.component.scss

```diff
@@ -1,10 +1,8 @@
+@use '../../../../libs/design-system/styles/vars' as vars;
```

Comment:
Inside the apps it should be possible to just include the file as `@use 'vars'`
The style preprocessor should already be configured to look for the design system style files